clang/test/Driver/openacc-no-cir.c

```diff
@@ -0,0 +1,6 @@
+// RUN: not %clang -fopenacc %s 2>&1 | FileCheck %s -check-prefix=ERROR
+// RUN: not %clang -fclangir -fopenacc %s 2>&1 | FileCheck %s -check-prefix=NOERROR
+// RUN: not %clang -fopenacc -fclangir %s 2>&1 | FileCheck %s -check-prefix=NOERROR
+
+// ERROR: OpenACC directives will result in no runtime behavior, use -fclangir to enable runtime effect
+// NOERROR-NOT: OpenACC directives will result in no runtime behavior, use -fclangir to enable runtime effect
```

Comment:
Can you use `-Werror` instead of the negative check?

Reply:
Unfortunately not, driver warnings are... special and don't really pay attention to `Werror` since they happen "before" `Werror` has been processed.

I HAVE however modified the test run-lines in a way that are perhaps better, in that they don't try to link now, so hopefully they do a better job diagnosing.  I've also made the 'NOT' line a little less specific, so hopefully it'll not go stale as easy.